scRNA-seq/04-dimension_reduction_scRNA.Rmd

```diff
@@ -230,7 +230,7 @@ assayNames(normalized_sce)
 rm(hodgkins_sce, filtered_sce)
 ```
 
-![**Status of the current dataset**](diagrams/tag-based_3.png)
+![**Status of the current dataset**](diagrams/tag-based_4.png)
```

Comment:
I think for this one we need a new diagram with Cell Ranger swapped in for Alevin, since that is what we are using for this notebook. I might have it go beyond to the "downstream analysis" as we have in a later slide in that set?

Reply:
Really good point. I went ahead and made a new roadmap to use for the cellranger based notebooks for day 2 and added those diagrams to the folder and updated them in the notebooks. There is one that can be added in to the cell clustering notebook when that is complete as well, but let me know if I am missing any major steps. I thought it might be nice to include dimensionality reduction and cell clustering as individual steps, but let me know if you would prefer it to be just one bullet and labeled "downstream analysis" instead.